plugins/grasper/CMakeLists.txt

```diff
@@ -6,15 +6,18 @@ if( QHULL_FOUND AND QHULL_INCLUDE_DIR )
 endif()
 
 add_library(grasper SHARED grasper.cpp graspermodule.cpp grasperplanner.cpp  plugindefs.h)
+target_link_libraries(grasper PRIVATE boost_assertion_failed)
 
 if ( QHULL_FOUND )
   add_definitions(-DQHULL_FOUND)
   if( QHULL_USE_REENTRANT )
     add_definitions(-DQHULL_USE_REENTRANT)
+    target_link_libraries(grasper PUBLIC libopenrave qhull_r)
```

Comment:
@cielavenir instead of having `target_link_libraries` three times. Why not assign the qhull library that will be used in a `USING_QHULL_LIBRARY` variable? and just use that within `target_link_libraries`?

Reply:
done